src/features/vars-schema/env-vars.ts

```diff
@@ -343,6 +343,38 @@ export const DEVICE_TYPE_SPECIFIC_CONFIG_VAR_PROPERTIES: Array<{
 			},
 		},
 	),
+	...getDefinitionWithMinimumSupervisorVersion(
+		{
+			'14.4.7': [
+				'raspberrypi4-64',
+				'raspberrypi400-64',
+				'raspberrypicm4-ioboard',
+			],
+		},
+		{
+			'BALENA_HOST_CONFIG_hdmi_force_hotplug:1': {
```

Comment:
Have you tested this to confirm whether any part of the stack might freak out by the `:`?

Reply:
I am working on this, the supervisor freaks out, but I will create a new version that accepts it, hence the limiting by version. I need to see if the dashboard will accept the change.

Reply:
Ok, just tested, I need to change the `device_config_variable` resource, because I'm getting `Error!Request error: Configuration variable names can only contain alphanumeric characters and underscores.`